app/views/hub/efile_submissions/_log.html.erb

```diff
@@ -11,8 +11,19 @@
           <%= error.message %>
         </div>
     <% end %>
-    <% if transition.metadata["indicators"].present? %>
-      <div> Fraud indicators: <%= transition.metadata["indicators"].map(&:humanize).join(", ") %></div>
+    <% if transition.to_state.in? ["bundling", "fraud_hold"] %>
```

Comment:
every client will be assigned a fraud score, but some may not exceed the threshold to put them on hold. For example, if an international timezone is there but its the only indicator, their score would only be 20 points so they get passed to bundling.

Each submission will only ever be in bundling OR fraud hold, because we'll create another submission when they resubmit.